// @fluentui/react-datepicker-compat/lib-commonjs/components/DatePicker/index.js
"use strict";
Object.defineProperty(exports, "__esModule", {
    value: true
});
function _export(target, all) {
    for(var name in all)Object.defineProperty(target, name, {
        enumerable: true,
        get: all[name]
    });
}
_export(exports, {
    DatePicker: function() {
        return _DatePicker.DatePicker;
    },
    datePickerClassNames: function() {
        return _useDatePickerStylesstyles.datePickerClassNames;
    },
    defaultDatePickerErrorStrings: function() {
        return _defaults.defaultDatePickerErrorStrings;
    },
    defaultDatePickerStrings: function() {
        return _defaults.defaultDatePickerStrings;
    },
    renderDatePicker_unstable: function() {
        return _renderDatePicker.renderDatePicker_unstable;
    },
    useDatePickerStyles_unstable: function() {
        return _useDatePickerStylesstyles.useDatePickerStyles_unstable;
    },
    useDatePicker_unstable: function() {
        return _useDatePicker.useDatePicker_unstable;
    }
});
const _defaults = require("./defaults");
const _DatePicker = require("./DatePicker");
const _renderDatePicker = require("./renderDatePicker");
const _useDatePicker = require("./useDatePicker");
const _useDatePickerStylesstyles = require("./useDatePickerStyles.styles");
