// @fluentui/react-calendar-compat/lib-commonjs/utils/dateFormatting/index.js
"use strict";
Object.defineProperty(exports, "__esModule", {
    value: true
});
function _export(target, all) {
    for(var name in all)Object.defineProperty(target, name, {
        enumerable: true,
        get: all[name]
    });
}
_export(exports, {
    DEFAULT_CALENDAR_STRINGS: function() {
        return _dateFormattingdefaults.DEFAULT_CALENDAR_STRINGS;
    },
    DEFAULT_DATE_FORMATTING: function() {
        return _dateFormattingdefaults.DEFAULT_DATE_FORMATTING;
    },
    DEFAULT_DATE_GRID_STRINGS: function() {
        return _dateFormattingdefaults.DEFAULT_DATE_GRID_STRINGS;
    },
    formatDay: function() {
        return _dateFormattingdefaults.formatDay;
    },
    formatMonth: function() {
        return _dateFormattingdefaults.formatMonth;
    },
    formatMonthDayYear: function() {
        return _dateFormattingdefaults.formatMonthDayYear;
    },
    formatMonthYear: function() {
        return _dateFormattingdefaults.formatMonthYear;
    },
    formatYear: function() {
        return _dateFormattingdefaults.formatYear;
    }
});
const _dateFormattingdefaults = require("./dateFormatting.defaults");
