// @fluentui/react-calendar-compat/lib-commonjs/utils/index.js
"use strict";
Object.defineProperty(exports, "__esModule", {
    value: true
});
function _export(target, all) {
    for(var name in all)Object.defineProperty(target, name, {
        enumerable: true,
        get: all[name]
    });
}
_export(exports, {
    DAYS_IN_WEEK: function() {
        return _constants.DAYS_IN_WEEK;
    },
    DEFAULT_CALENDAR_STRINGS: function() {
        return _dateFormatting.DEFAULT_CALENDAR_STRINGS;
    },
    DEFAULT_DATE_FORMATTING: function() {
        return _dateFormatting.DEFAULT_DATE_FORMATTING;
    },
    DEFAULT_DATE_GRID_STRINGS: function() {
        return _dateFormatting.DEFAULT_DATE_GRID_STRINGS;
    },
    DURATION_1: function() {
        return _animations.DURATION_1;
    },
    DURATION_2: function() {
        return _animations.DURATION_2;
    },
    DURATION_3: function() {
        return _animations.DURATION_3;
    },
    DURATION_4: function() {
        return _animations.DURATION_4;
    },
    DateRangeType: function() {
        return _constants.DateRangeType;
    },
    DayOfWeek: function() {
        return _constants.DayOfWeek;
    },
    EASING_FUNCTION_1: function() {
        return _animations.EASING_FUNCTION_1;
    },
    EASING_FUNCTION_2: function() {
        return _animations.EASING_FUNCTION_2;
    },
    FADE_IN: function() {
        return _animations.FADE_IN;
    },
    FADE_OUT: function() {
        return _animations.FADE_OUT;
    },
    FirstWeekOfYear: function() {
        return _constants.FirstWeekOfYear;
    },
    MonthOfYear: function() {
        return _constants.MonthOfYear;
    },
    SLIDE_DOWN_IN20: function() {
        return _animations.SLIDE_DOWN_IN20;
    },
    SLIDE_DOWN_OUT20: function() {
        return _animations.SLIDE_DOWN_OUT20;
    },
    SLIDE_LEFT_IN20: function() {
        return _animations.SLIDE_LEFT_IN20;
    },
    SLIDE_RIGHT_IN20: function() {
        return _animations.SLIDE_RIGHT_IN20;
    },
    SLIDE_UP_IN20: function() {
        return _animations.SLIDE_UP_IN20;
    },
    SLIDE_UP_OUT20: function() {
        return _animations.SLIDE_UP_OUT20;
    },
    TRANSITION_ROW_DISAPPEARANCE: function() {
        return _animations.TRANSITION_ROW_DISAPPEARANCE;
    },
    TimeConstants: function() {
        return _constants.TimeConstants;
    },
    addDays: function() {
        return _dateMath.addDays;
    },
    addMonths: function() {
        return _dateMath.addMonths;
    },
    addWeeks: function() {
        return _dateMath.addWeeks;
    },
    addYears: function() {
        return _dateMath.addYears;
    },
    compareDatePart: function() {
        return _dateMath.compareDatePart;
    },
    compareDates: function() {
        return _dateMath.compareDates;
    },
    findAvailableDate: function() {
        return _dateGrid.findAvailableDate;
    },
    focusAsync: function() {
        return _focus.focusAsync;
    },
    formatDay: function() {
        return _dateFormatting.formatDay;
    },
    formatMonth: function() {
        return _dateFormatting.formatMonth;
    },
    formatMonthDayYear: function() {
        return _dateFormatting.formatMonthDayYear;
    },
    formatMonthYear: function() {
        return _dateFormatting.formatMonthYear;
    },
    formatYear: function() {
        return _dateFormatting.formatYear;
    },
    getBoundedDateRange: function() {
        return _dateGrid.getBoundedDateRange;
    },
    getDatePartHashValue: function() {
        return _dateMath.getDatePartHashValue;
    },
    getDateRangeArray: function() {
        return _dateMath.getDateRangeArray;
    },
    getDayGrid: function() {
        return _dateGrid.getDayGrid;
    },
    getEndDateOfWeek: function() {
        return _dateMath.getEndDateOfWeek;
    },
    getMonthEnd: function() {
        return _dateMath.getMonthEnd;
    },
    getMonthStart: function() {
        return _dateMath.getMonthStart;
    },
    getStartDateOfWeek: function() {
        return _dateMath.getStartDateOfWeek;
    },
    getWeekNumber: function() {
        return _dateMath.getWeekNumber;
    },
    getWeekNumbersInMonth: function() {
        return _dateMath.getWeekNumbersInMonth;
    },
    getYearEnd: function() {
        return _dateMath.getYearEnd;
    },
    getYearStart: function() {
        return _dateMath.getYearStart;
    },
    isInDateRangeArray: function() {
        return _dateMath.isInDateRangeArray;
    },
    isRestrictedDate: function() {
        return _dateGrid.isRestrictedDate;
    },
    setMonth: function() {
        return _dateMath.setMonth;
    }
});
const _animations = require("./animations");
const _constants = require("./constants");
const _dateFormatting = require("./dateFormatting");
const _dateGrid = require("./dateGrid");
const _dateMath = require("./dateMath");
const _focus = require("./focus");
