// @fluentui/react-calendar-compat/lib-commonjs/utils/dateMath/index.js
"use strict";
Object.defineProperty(exports, "__esModule", {
    value: true
});
function _export(target, all) {
    for(var name in all)Object.defineProperty(target, name, {
        enumerable: true,
        get: all[name]
    });
}
_export(exports, {
    addDays: function() {
        return _dateMath.addDays;
    },
    addMonths: function() {
        return _dateMath.addMonths;
    },
    addWeeks: function() {
        return _dateMath.addWeeks;
    },
    addYears: function() {
        return _dateMath.addYears;
    },
    compareDatePart: function() {
        return _dateMath.compareDatePart;
    },
    compareDates: function() {
        return _dateMath.compareDates;
    },
    getDatePartHashValue: function() {
        return _dateMath.getDatePartHashValue;
    },
    getDateRangeArray: function() {
        return _dateMath.getDateRangeArray;
    },
    getEndDateOfWeek: function() {
        return _dateMath.getEndDateOfWeek;
    },
    getMonthEnd: function() {
        return _dateMath.getMonthEnd;
    },
    getMonthStart: function() {
        return _dateMath.getMonthStart;
    },
    getStartDateOfWeek: function() {
        return _dateMath.getStartDateOfWeek;
    },
    getWeekNumber: function() {
        return _dateMath.getWeekNumber;
    },
    getWeekNumbersInMonth: function() {
        return _dateMath.getWeekNumbersInMonth;
    },
    getYearEnd: function() {
        return _dateMath.getYearEnd;
    },
    getYearStart: function() {
        return _dateMath.getYearStart;
    },
    isInDateRangeArray: function() {
        return _dateMath.isInDateRangeArray;
    },
    setMonth: function() {
        return _dateMath.setMonth;
    }
});
const _dateMath = require("./dateMath");
